packages/manager/apps/pci-load-balancer/src/pages/detail/listeners/l7/rules/L7Rules.page.tsx

```diff
@@ -0,0 +1,97 @@
+import { Suspense } from 'react';
+import { Outlet, useParams } from 'react-router-dom';
+import {
+  OsdsBreadcrumb,
+  OsdsIcon,
+  OsdsLink,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { useTranslation } from 'react-i18next';
+import { useProject } from '@ovh-ux/manager-pci-common';
+import { Headers, useProjectUrl } from '@ovh-ux/manager-react-components';
+import { useListener } from '@/api/hook/useListener';
+
+export default function L7RulesPage() {
+  const { t: tPciCommon } = useTranslation('pci-common');
+  const { t: tL7Policies } = useTranslation('octavia-load-balancer-l7');
+  const { t } = useTranslation('l7/rules');
+  const hrefProject = useProjectUrl('public-cloud');
+  const { t: tLoadBalancer } = useTranslation('octavia-load-balancer');
+  const { t: tListener } = useTranslation('octavia-load-balancer-listeners');
+  const { projectId, loadBalancerId, region, listenerId } = useParams();
+  const { data: project } = useProject();
+  const { data: listener } = useListener({
+    projectId,
+    region,
+    loadBalancerId,
+    listenerId,
+  });
+  return (
+    <>
+      <OsdsBreadcrumb
+        items={[
+          {
+            href: hrefProject,
+            label: project.description,
+          },
+          {
+            href: `${hrefProject}/octavia-load-balancer`,
```

Comment:
since it's not an external link shouldn't you use useHref for generating breadcrumb's links ?